cmd/talosctl/cmd/talos/events.go

```diff
@@ -24,7 +31,21 @@ var eventsCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		return WithClient(func(ctx context.Context, c *client.Client) error {
 			w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
-			fmt.Fprintln(w, "NODE\tEVENT\tMESSAGE")
+			fmt.Fprintln(w, "NODE\tID\tEVENT\tOBJECT\tMESSAGE")
```

Comment:
I can't figure out what `OBJECT` is here.

Reply:
not sure if `OBJECT` is the best name, but looks like all the events have some kind of name/identitifer:

* phase/sequence/task name
* service name

So we could have provided some filtering in the future on it.

Reply:
I thought that that was what `EVENT` is. I believe its the name.

Reply:
`EVENT` right now is event type ID. 

Example output with this PR:

```
 talosctl events --tail=15
NODE         EVENT                                     ID                     OBJECT     MESSAGE
172.20.0.1   talos/runtime/machine.ServiceStateEvent   bs1o9kq2qblc738ovma0   bootkube   PREPARING: Running pre state
172.20.0.1   talos/runtime/machine.ServiceStateEvent   bs1o9kq2qblc738ovmag   bootkube   PREPARING: Creating service runner
172.20.0.1   talos/runtime/machine.ServiceStateEvent   bs1o9kq2qblc738ovmb0   bootkube   RUNNING: Started task bootkube (PID 2231) for container bootkube
172.20.0.1   talos/runtime/machine.SequenceEvent   bs1o9uq2qblc738ovmgg   boot   STOP
172.20.0.1   talos/runtime/machine.ServiceStateEvent   bs1oa822qblc738ovmh0   bootkube   FINISHED: Service finished successfully
```

Reply:
Ahh ok that makes sense. Pretty cool. What about `SOURCE`?

Reply:
makes perfect sense to me, I will update!